app-modules/assistant/src/Providers/AssistantServiceProvider.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Assist\Assistant\Providers;
+
+use Filament\Panel;
+use Assist\Assistant\AssistantPlugin;
+use Illuminate\Support\ServiceProvider;
+use Assist\Assistant\Models\AssistantChat;
+use Assist\Assistant\Models\AssistantChatMessage;
+use Assist\Authorization\AuthorizationRoleRegistry;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use Assist\Assistant\Services\AIInterface\AzureOpenAI;
+use Assist\Authorization\AuthorizationPermissionRegistry;
+use Assist\Assistant\Services\AIInterface\Contracts\AIInterface;
+
+class AssistantServiceProvider extends ServiceProvider
+{
+    public function register()
+    {
+        Panel::configureUsing(fn (Panel $panel) => $panel->plugin(new AssistantPlugin()));
+
+        $this->app->bind(AIInterface::class, AzureOpenAI::class);
+    }
+
+    public function boot()
+    {
+        Relation::morphMap([
+            'assistant-chat' => AssistantChat::class,
```

Comment:
I believe other models are using `snake_case`, not `kebab-case` (as seen in `MorphServiceProvider`)

Reply:
Ah, yes, good catch. Updated.